docker/1.13/Dockerfile.eia

```diff
@@ -1,6 +1,5 @@
 FROM ubuntu:16.04
 LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
-LABEL com.amazonaws.sagemaker.capabilities.multi-models=true
```

Comment:
1. not clear what this change has to do with ping
2. does this mean we are removing the mme feature from this version? that's a breaking change.

Reply:
Hosting decided not to support eia and gpu at the moment, thus removing the labels. (Same as below.)